pkg/render/guardian.go

```diff
@@ -52,18 +52,22 @@ import (
 
 // The names of the components related to the Guardian related rendered objects.
 const (
-	GuardianName                   = "tigera-guardian"
+	GuardianName                   = "guardian"
 	GuardianNamespace              = common.CalicoNamespace
 	GuardianServiceAccountName     = GuardianName
-	GuardianClusterRoleName        = GuardianName
-	GuardianClusterRoleBindingName = GuardianName
+	GuardianClusterRoleName        = "calico-guardian"
+	GuardianClusterRoleBindingName = "calico-guardian"
 	GuardianDeploymentName         = GuardianName
-	GuardianServiceName            = "tigera-guardian"
-	GuardianVolumeName             = "tigera-guardian-certs"
-	GuardianSecretName             = "tigera-managed-cluster-connection"
-	GuardianTargetPort             = 8080
-	GuardianPolicyName             = networkpolicy.TigeraComponentPolicyPrefix + "guardian-access"
-	GuardianKeyPairSecret          = "guardian-key-pair"
+
+	// GuardianContainerName name is the name of the container running guardian. It's named `tigera-guardian`, instead
+	// of `guardian` so that the API for the container overrides don't have to change (`tigera-guardian` is a legacy name).
+	GuardianContainerName = "tigera-guardian"
+	GuardianServiceName   = "guardian"
+	GuardianVolumeName    = "guardian-certs"
+	GuardianSecretName    = "tigera-managed-cluster-connection"
```

Comment:
Any reason this can't be `guardian-managed-cluster-connection` or `calico-managed-cluster-connection` or just `managed-cluster-connection`?

Reply:
Not asking you to do this now, but seems like it should probably be a SecretRef on the ManagedClusterConnection resource rather than hard-coded.